content/master/guides/mrd-activation-policies.md

```diff
@@ -0,0 +1,689 @@
+```
+
+## Validation and testing
+
+### Check activation status
+
+Verify your policies work:
+
+```shell
+# List all activation policies
+kubectl get mrap
+
+# Check specific policy status
+kubectl describe mrap my-policy
+
+# See which MRDs are currently active
+kubectl get mrds --field-selector spec.state=Active
+
+# Count active MRDs by provider
+kubectl get mrds -l crossplane.io/provider=provider-aws --field-selector spec.state=Active | wc -l
+```
+
+### Test resource creation
+
+Verify activated resources work:
+
+```shell
+# Try creating a managed resource
+kubectl apply -f - <<EOF
+apiVersion: s3.aws.crossplane.io/v1alpha1
+kind: Bucket
+metadata:
+  name: test-activation
+spec:
+  forProvider:
+    region: us-east-1
+EOF
+
+# Check if the resource was created successfully
+kubectl get bucket test-activation
+kubectl describe bucket test-activation
+```
+
+### Policy debugging
+
+Common issues and solutions:
+
+```shell
+# Check if MRD exists but isn't active
+kubectl get mrd my-resource.provider.example.com
+kubectl describe mrd my-resource.provider.example.com
+
+# Verify policy is matching  
+kubectl get mrap my-policy -o yaml | grep -A20 status
+
+# Look for controller errors
+kubectl logs -n crossplane-system deployment/crossplane | grep -i mrd
+```
+
+## Performance considerations
+
+### Activation overhead
+
+Each activation creates a CRD, which has resource overhead:
+
+```yaml
+# Efficient - activates exactly what's needed
+spec:
+  activations:
+  - instances.ec2.aws.crossplane.io
+  - buckets.s3.aws.crossplane.io
+
+# Less efficient - activates everything
+spec:
+  activations:
+  - "*.aws.crossplane.io"  # Could be 200+ resources
+```
+
+### Policy consolidation
+
+Multiple small policies vs. large policies:
+
+{{< tabs >}}
+{{< tab "Multiple Small Policies (Recommended)" >}}
+```yaml
+# Easier to manage and understand
+---
+apiVersion: apiextensions.crossplane.io/v1alpha1
+kind: ManagedResourceActivationPolicy
+metadata:
+  name: compute-resources
+spec:
+  activations:
+  - "*.ec2.aws.crossplane.io"
+  - "*.compute.gcp.crossplane.io"
+
+---
+apiVersion: apiextensions.crossplane.io/v1alpha1
+kind: ManagedResourceActivationPolicy
+metadata:
+  name: storage-resources  
+spec:
+  activations:
+  - "*.s3.aws.crossplane.io"
+  - "*.storage.gcp.crossplane.io"
+```
+{{< /tab >}}
+
+{{< tab "Single Large Policy" >}}
+```yaml
+# Harder to manage but fewer resources
+apiVersion: apiextensions.crossplane.io/v1alpha1
+kind: ManagedResourceActivationPolicy
+metadata:
+  name: all-resources
+spec:
+  activations:
+  - "*.ec2.aws.crossplane.io"
+  - "*.compute.gcp.crossplane.io"
+  - "*.s3.aws.crossplane.io"
+  - "*.storage.gcp.crossplane.io"
+  # ... many more activations
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+## Policy lifecycle management
+
+<!-- vale Google.Headings = NO -->
+### GitOps workflow
+<!-- vale Google.Headings = YES -->
+
+Store activation policies in Git for proper change management:
```

Comment:
This feels low value. "Use GitOps" could be true for any component of Crossplane. I don't think we need a specific callout for MRAPs.